src/BulletCollision/NarrowPhaseCollision/btGjkEpa2.cpp

```diff
@@ -41,21 +41,38 @@ namespace gjkepa2_impl
 
 	/* GJK	*/ 
 #define GJK_MAX_ITERATIONS	128
-#define GJK_ACCURARY		((btScalar)0.0001)
-#define GJK_MIN_DISTANCE	((btScalar)0.0001)
-#define GJK_DUPLICATED_EPS	((btScalar)0.0001)
+
+#ifdef BT_USE_DOUBLE_PRECISION
+	#define GJK_ACCURARY		((btScalar)1e-12)
```

Comment:
Would this be a good time to fix the spelling? `GJK_ACCURARY` -> `GJK_ACCURACY` ?